src/main/resources/db/migration/0350/V351__SystemMetrics.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE accelerator.system_metrics(
+    id INTEGER PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    name TEXT NOT NULL,
+    type_id INTEGER NOT NULL REFERENCES accelerator.metric_types,
+    component_id INTEGER NOT NULL REFERENCES accelerator.metric_components,
+    description TEXT,
+    reference TEXT COLLATE natural_numeric NOT NULL
+);
+
+CREATE TABLE accelerator.report_system_metrics(
+    report_id BIGINT NOT NULL REFERENCES accelerator.reports ON DELETE CASCADE,
+    system_metric_id INTEGER NOT NULL REFERENCES accelerator.system_metrics,
+    target INTEGER,
+    system_value INTEGER,
+    system_time TIMESTAMP WITH TIME ZONE,
+    override_value INTEGER,
+    notes TEXT,
+    internal_comment TEXT,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL,
+
+    PRIMARY KEY (report_id, system_metric_id),
+
+    CONSTRAINT system_time CHECK ((
+        system_value IS NULL AND
+        system_time IS NULL
+    ) OR (
+        system_value IS NOT NULL AND
+        system_time IS NOT NULL
+    ))
+);
+
+CREATE INDEX ON accelerator.report_system_metrics(system_metric_id);
+CREATE INDEX ON accelerator.report_system_metrics(report_id, system_metric_id);
```

Comment:
Graphite's comment here is correct; this index is redundant because `PRIMARY KEY (report_id, system_metric_id)` creates a unique index already.